docs/howtoguides/get_token_and_attach.md

```diff
@@ -35,3 +46,24 @@ Enable services with: pro enable <service>
 Once the Ubuntu Pro Client is attached to your Ubuntu Pro account, you can use
 it to activate various services, including: access to ESM packages, Livepatch,
 FIPS, and CIS. Some features are specific to certain LTS releases.
+
+## Control of auto-enabled services
+
+Your subscription controls which services are available to you as shown and
+controlled on the [Ubuntu Pro Dashboard](https://ubuntu.com/pro/dashboard).
+
+Recommended services are auto-enabled when attaching a system. Which services
+are considered recommended can, for some subscriptions, be controlled on the
+[Ubuntu Pro Dashboard](https://ubuntu.com/pro/dashboard) by toggling those.
+
+![Toggling recommended services in the Pro Dashboard](pro-dashboard-service-toggles.png)
+
+In case you can not control your subscription (depends on the subscription type
+and your users permissions related to it). Or if you do not want to change the
```

Comment:
```suggestion
and the user permissions granted by it). Or if you do not want to change the
```

Reply:
Umm, no :-)

Subscriptions differ and in them then users differ:
- free one - just defaults no permission to change via the web UI
- real subscription has some user management, you might have the permission to change things or only to see them

Your suggestion implies that the subscription would grant any permissions. That is not what I was saying. I wanted to avoid 500 more words but make it clear that (depending on complex conditions) you might or might not be able to see or tweak them.